io/jvm/src/main/scala/fs2/io/ioplatform.scala

```diff
@@ -144,4 +146,25 @@ private[fs2] trait ioplatform extends iojvmnative {
     }
   }
 
+  private lazy val vtExecutor: Option[ExecutionContext] = {
```

Comment:
Since this is private code, instead of `Option` we can just use `null`, that will be a bit more efficient than the `fold`.